### ISSUE TITLE
Failed to create order

### ISSUE THREAD

**Issue by aelvan** [elleracompany/commerce-klarna-checkout#68]
i'm seeing a lot of these kinds of errors in the klarna-checkout.log:

```
2021-11-24 20:39:35 Failed to create order 27506. Klarna responded with 201: Created
2021-11-25 10:51:45 Failed to create order 27555. Klarna responded with 201: Created
2021-11-25 10:59:38 Failed to create order 12587. Klarna responded with 201: Created
2021-11-25 11:00:58 Failed to create order 12587. Klarna responded with 201: Created
2021-11-25 11:01:32 Failed to create order 12587. Klarna responded with 201: Created
2021-11-25 11:53:03 Failed to create order 27559. Klarna responded with 201: Created
```

Is this an error I need to care about, or is it to be expected? :)

Still on 3.0.3 due to #67 .

**Comment by jellingsen** [comment]
As long as the orders are created this should be no problem. Klarna is responding correctly. Seems like a bug in the log message.

I'm trying to clear the schedule for an update, I'll fix this and #67 as soon as I can.

**Comment by jellingsen** [comment]
Hi again,

This was fixed in this commit: https://github.com/elleracompany/commerce-klarna-checkout/commit/f97d0569db5764f5c5973907cc024fdf7134a4b9 

I'll get on #67 so you can update to a version without this bug.
